tower-balance/examples/demo.rs

```diff
@@ -305,3 +323,31 @@ where
         Ok(Async::Ready(None))
     }
 }
+
+pub struct Errify<T>(T);
```

Comment:
is this something we will always need? or is it just for compat between other services that do not yet use `Box<Error>`?

Reply:
It should go away once everything is `Box<Error>`.